tests/test_open.py

```diff
@@ -0,0 +1,76 @@
+########################################################################
+#
+#       Created: May 26, 2021
```

Comment:
This has been created later, right? ;-). At any rate, I don't think this field has much sense, and I'd remove it from all files.